src/main/java/io/confluent/connect/hdfs/hive/HiveMetaStore.java

```diff
@@ -50,7 +50,9 @@ public HiveMetaStore(Configuration conf, HdfsSinkConnectorConfig connectorConfig
     HiveConf hiveConf = new HiveConf(conf, HiveConf.class);
     String hiveConfDir = connectorConfig.getString(HdfsSinkConnectorConfig.HIVE_CONF_DIR_CONFIG);
     String hiveMetaStoreURIs = connectorConfig.getString(HdfsSinkConnectorConfig.HIVE_METASTORE_URIS_CONFIG);
-    hiveConf.addResource(new Path(hiveConfDir, "hive-site.xml"));
+    if (!hiveConfDir.equals("")) {
+      hiveConf.addResource(new Path(hiveConfDir, "/hive-site.xml"));
+    }
```

Comment:
Minor: Validate that hive-site.xml exists and print a warning if it doesn't? Looks like this can be a common mistake.